Nitrox.Assets.Subnautica/LanguageFiles/en.json

```diff
@@ -1,9 +1,14 @@
 {
     "Nitrox_AddServer": "Add Server",
-    "Nitrox_AddServerAdd": "Add",
-    "Nitrox_AddServerHost": "Host:",
-    "Nitrox_AddServerName": "Name:",
-    "Nitrox_AddServerPort": "Port:",
+    "Nitrox_AddServer_NameDescription": "Name:",
+    "Nitrox_AddServer_AddressDescription": "Host:",
+    "Nitrox_AddServer_PortDescription": "Port:",
+    "Nitrox_AddServer_NamePlaceholder": "Enter a name for the server",
+    "Nitrox_AddServer_AddressPlaceholder": "Enter the IP or address of the server",
+    "Nitrox_AddServer_PortPlaceholder": "Enter the numeric port of the server",
+    "Nitrox_AddServer_Confirm": "Create server",
+    "Nitrox_AddServer_Back": "Back",
+    "Nitrox_AddServer_InvalidInput": "You input is invalid. Please check if all text boxes are filled out.",
```

Comment:
Nit: "You" -> "Your"

Per [UX guidelines](https://www.nngroup.com/articles/error-message-guidelines/), we should try to avoid blaming the user with error messages. This will help the mod feel more approachable, especially for people who aren't as familiar with server IPs.

Suggested: "Text boxes should not be empty."

Reply:
"You" in this regard is not grammatically correct and therefor would not make much sense in the English language. I understand wanting the mod to feel more approachable however if you wanted to do so it should be something like "The input is invalid"